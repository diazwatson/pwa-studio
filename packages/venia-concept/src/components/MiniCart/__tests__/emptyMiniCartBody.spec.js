import React from 'react';
import ShallowRenderer from 'react-test-renderer/Shallow';

import EmptyMiniCartBody from '../emptyMiniCartBody';

const renderer = new ShallowRenderer();

test('renders a "no items" message', () => {
    const tree = renderer.render(<EmptyMiniCartBody />);

    expect(tree).toMatchSnapshot();
});
