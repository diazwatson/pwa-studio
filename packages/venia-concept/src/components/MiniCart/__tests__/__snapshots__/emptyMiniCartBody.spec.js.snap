// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a "no items" message 1`] = `
<div>
  <h3>
    There are no items in your shopping cart
  </h3>
  <Classify(Trigger)>
    <span>
      Continue Shopping
    </span>
  </Classify(Trigger)>
</div>
`;
